### ISSUE TITLE
Hero Banner Accessibility on images - ITSADSSD-31873

### ISSUE THREAD

**Issue by Bcrunk** [uq-its-ss/design-system#203]
The example Hero Banner with Image and possibly the current structural code is returning some issues. 

![image](https://user-images.githubusercontent.com/8379323/116964220-54cf5580-acee-11eb-96f9-f56d4b40e41d.png)

Review the outstanding issues: 

- [ ] Critical:  Element does not have an alt attribute
- [ ] Serious: aria-label attribute does not exist or is empty
- [ ] Serious: aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
- [ ] Serious: Element has no title attribute
- [ ] Minor: Element's default semantics were not overridden with role="none" or role="presentation"
